lib/stemcell/launcher.rb

```diff
@@ -291,5 +297,44 @@ def try_file(opt="")
         File.read(File.expand_path(opt)) rescue opt
     end
 
+    MAX_ATTEMPT = 3
+    def initial_retry_sec
+      return 1
+    end
```

Comment:
If this is a method only so we can stub it in test, I'd prefer we use something like timecop or monkeypatch sleep in test and leave this as a constant.